src/components/MovieReviewItem/index.js

```diff
@@ -27,11 +33,24 @@ const MovieReviewItem = ({
             {/*}*/}
             <div className="col-12 card border-0 bg-light">
                 <div className="card-body d-flex justify-content-between">
-                    <div>
-                        <h5 className="card-title p-1">User: {review.postedBy.username}</h5>
+                {location.pathname === '/profile/s/reviews' && <div className="row" onClick={() => onClickNavigate(review.movie)}>
+                        <div className="col-4">
```

Comment:
I'm not sure why, but the grid system is not working correctly. The poster size is not the same in each review. I suggest removing all the card-related classes and using normal divs.

Reply:
Can we get on a call to discuss this?